syntax = "proto3";

package Messages;

import "google/protobuf/any.proto";

message UserCommandToStart
{}
